ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml

```diff
@@ -159,6 +159,30 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-azure
   timeout: 5h0m0s
+- always_run: false
+  as: node-density-heavy-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
```

Comment:
Can you add infra instance size too
`      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: Standard_E16s_v3`

Reply:
Qiujie told me: 

> You don't need these two, that's for infra on large clusters like 120. But that is not related to the failures you see.
>
>     OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: Standard_E16s_v3
>
>     SET_ENV_BY_PLATFORM: custom
>    
> Your failures are installation issues.

Reply:
Standard_E16s_v3 should not be needed on a 24 nodes. With 120 nodes the prometheus has more pressure so we have to use a bigger infra node.